spec/controllers/api/devices/devices_controller_show_spec.rb

```diff
@@ -4,6 +4,15 @@
 
   include Devise::TestHelpers
 
+  let(:user) { FactoryGirl.create(:user) }
+
   describe '#show' do
+    it 'handles deviceless requests' do
+      user.update_attributes(device: nil)
+      sign_in user
+      get :show, {}, format: :json
+      expect(json[:error]).to eq("add device to account")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.